packages/calling-stateful-client/src/CallDeclarativeCommon.ts

```diff
@@ -98,6 +98,11 @@ export abstract class ProxyCallCommon implements ProxyHandler<CallCommon> {
                 this._context.setIsCaptionActive(target.id, true);
                 return ret;
               },
+              stopCaptions: async (...args: Parameters<TeamsCaptionsCallFeature['startCaptions']>) => {
+                const ret = await captionsFeature.startCaptions(...args);
```

Comment:
Should this line be stopCaptions?

Reply:
Nice catch! Actually stopCaptions doesn't have any parameter